### DISCUSSION TITLE
How do I edit a page in markdown mode?

### DISCUSSION THREAD

**Discussion by smpetersgithub** [GitbookIO/community#781] [1. How to] [answered]
How do I edit a page in full Markdown mode? In GitHub, I go to Edit File -> Edit in Place, and I can edit using the Markdown language.

**Comment by addisonschultz** [comment, marked as answer]
Do you have your space synced with GitHub? https://docs.gitbook.com/integrations/git-sync

You will be able to edit markdown files after you set this up!
